controllers/getCats.js

```diff
@@ -0,0 +1,22 @@
+function getCats() {
+    request('https://api.thecatapi.com/v1/breeds', {
+            'x-api-key': process.env.API_KEY_CATBREEDS
+        },
+        function (error, response, body) {
+            if (!error && response.statusCode == 200) {
+                console.log('statusCode external Cat API request:', response && response.statusCode);
+                let catBreedsJSON = JSON.parse(body);
+                for (let breed of catBreedsJSON) {
+                    catBreeds.push(breed.name);
+                }
+                // then call getInformation()
+                getInformation();
+            } else {
+                console.log('error', error, response && response.statusCode);
+                catBreeds = null;
+                getInformation();
```

Comment:
If there's still time later on, we could add an issue to rewrite this part using promises, so there's no need stacking callbacks to first import the dogbreeds, catbreeds and user data before rendering the page, but for now this is fine.